docs/code-generation/generate-hcl.md

```diff
@@ -11,43 +11,40 @@ next:
   link: '/code-generation/generate-file'
 ---
 
-# HCL Code Generation
+# Generating HCL Code with Terramate
```

Comment:
Okay. 

Although a more explicit search like "how to generate HCL code with Terramate" might enhance search engine optimization to direct users here, I fully understand the rationale behind excluding "Terramate" to avoid redundancy.